PhinanceManager_REST/Controllers/IncomeCategoryController.cs

```diff
@@ -10,22 +10,22 @@
 namespace PhinanceManager_REST.Controllers
 {
     [ApiController]
-    [Route("phinancemanager/incomecategory")]
+    [Route("api/incomecategories")]
```

Comment:
Income Categories mozna zrobić tak samo jak payments categories. Kategorię są utażsamione z przychodem. Więc mozna to wrzucić do controllera IncomesController. A jesli ten controller miałby jednak zostać to ścieżka powinna być "api/incomes/categories"